### PR TITLE
Add automerges for 8.0 branches in asp.net repos

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -770,6 +770,7 @@
         "https://github.com/dotnet/efcore/blob/release/3.1//**/*",
         "https://github.com/dotnet/efcore/blob/release/5.0//**/*",
         "https://github.com/dotnet/efcore/blob/release/6.0//**/*",
+        "https://github.com/dotnet/efcore/blob/release/7.0//**/*",
```
Comment (wtgodbe) [inline Maestro/subscriptions.json:773]: @ajcvickers I noticed auto-merges from 7.0 to main in EFCore had been missing

**Review by joperezr — APPROVED** [review]
Thanks for taking care of this @wtgodbe.

File: Maestro/subscriptions.json [hunk 2]
```diff
@@ -770,6 +770,7 @@
         "https://github.com/dotnet/efcore/blob/release/3.1//**/*",
```
Comment (mmitche) [inline Maestro/subscriptions.json:770]: You can remove the EOL branches while you're at it.

**Review by mmitche — APPROVED** [review]

File: Maestro/subscriptions.json [hunk 3]
```diff
@@ -767,12 +767,9 @@
     {
       "triggerPaths": [
         "https://github.com/dotnet/ef6/blob/release/6.4//**/*",
-        "https://github.com/dotnet/efcore/blob/release/3.1//**/*",
-        "https://github.com/dotnet/efcore/blob/release/5.0//**/*",
         "https://github.com/dotnet/efcore/blob/release/6.0//**/*",
         "https://github.com/dotnet/efcore/blob/release/7.0//**/*",
         "https://github.com/dotnet/razor-compiler/blob/release/6.0.2xx//**/*",
```
Comment (mmitche) [inline Maestro/subscriptions.json]: I can't recall...is the trailing comma going to be an issue?

Comment (wtgodbe) [reply Maestro/subscriptions.json]: Already removed it

**Review by mmitche — APPROVED** [review]

**Pull request merged**